modules/nginx/manifests/redirect.pp

```diff
@@ -0,0 +1,44 @@
+define nginx::redirect (
+    $web_domain		= hiera("web_domain"),
+    $web_domain_redirect_from = hiera("web_domain_redirect_from"),
+    $ssl_crt_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.crt",
+    $ssl_key_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.key",
+    $server_name        = $name,
```

Comment:
`server_name` sounds more something like `pp0`, `app1`, but I think this is more intended to the "tenant alias" in OAE terminology. Can you rename it to something like `tenant_alias`?